### DISCUSSION TITLE
Is it possible to search in all blog posts regardless of their status?

### DISCUSSION THREAD

**Discussion by t1t4nium** [datenstrom/community#879] [Ask a question] [answered]
Hello

Related to the topic "Is it possible to filter blog posts by status:all?", is there any way to search (with the search extension) in all blog posts regardless of their status?

The purpose is the same, I'm building a private page where I can see all my posts and where I would like to search in all blog posts.

By default, the search is performed only on public posts. I saw that I could filter by status, but only one.

Regards

**Comment by annaesvensson** [comment]
No, this is not possible at the moment. 

The use case you described sounds like a useful addition. I am glad that you provided some context, because then we have a better chance of understanding what people are doing and which functionality is missing. Let us check how much source code this would take to implement, as you know we aim to keep things light and dead simple, when I know more I'll get back to you.

**Comment by annaesvensson** [comment, marked as answer]
The [search extension](https://github.com/annaesvensson/yellow-search) now supports more filter methods. Once you're logged in with your user account, you can search with `status:all` in all pages regardless of the status. This probably does what you would expect, you get pages with every possible status, not just one specific status. Here's another example, to search in all blog posts use `folder:blog status:all`.

**Reply by t1t4nium** [reply]
Thank you! I will try it.

**Reply by pftnhr** [reply]
I have just tested it and it really does display all pages. Regardless of the layout. Chronologically mixed up but all of them.
